packages/core/src/dialects/ibmi/data-types.ts

```diff
@@ -1,4 +1,4 @@
-import type { Falsy } from '../../generic/falsy.js';
+import type { Falsy } from '@sequelize/utils';
```

Comment:
same in this file. It's coercing any value not just boolean and falsy values

As a result, it can be removed from the utils package

Reply:
Removed now